### ISSUE TITLE
SharePoint Adaptive Card - Dark Mode Color Issue

### ISSUE THREAD

**Issue by ahmedmahmoudR** [SharePoint/sp-dev-docs#7682]
### Target SharePoint environment

SharePoint Online

### What SharePoint development model, framework, SDK or API is this about?

💥 SharePoint Framework

### Developer environment

Windows

### What browser(s) / client(s) have you tested

- [X] 💥 Internet Explorer
- [X] 💥 Microsoft Edge
- [X] 💥 Google Chrome
- [X] 💥 FireFox
- [ ] 💥 Safari
- [ ] mobile (iOS/iPadOS)
- [ ] mobile (Android)
- [ ] not applicable
- [ ] other (enter in the "Additional environment details" area below)

### Describe the bug / error

Now i have issue with ImageCard-HybridWork-Checkin Card when I change viva Sharepoint portal to dark mode the 
 the card below has some color issues which makes the input hard to see.
how to fix that 

![This is an image](https://i.imgur.com/7FsCBJz.jpg)

### Steps to reproduce

1.
2.
3.

### Expected behavior

see black color with input

**Comment by ahmedmahmoudR** [comment]
Any Update about this issue ?

**Comment by dcashpeterson** [comment]
I took a look at this and it looks like the form elements are not changing their text based on the theme of the SharePoint site. From what I could tell in the Adaptive Card documentation there isn't a color property on any of the form elements.

**Comment by ahmedmahmoudR** [comment]
Hello All Thanks for support but is there any update with my issue ?

**Comment by micwilson911** [comment]
Hi @ahmedmahmoudR,

Thank you for bringing this to our attention. There was a service fix that was applied at the end of March for Dark Mode coloring. Would you mind checking to see if this still repos on your end as I'm not able to repo the issue without reverting back commits.

**Comment by dcashpeterson** [comment]
@micwilson911 I checked this in my environment and I can see a change. I used the Dark Yellow theme and the form fields look like this. 
![image](https://user-images.githubusercontent.com/45491456/183433798-1d7a9ae6-78b9-4a2b-8586-645819a3aacd.png)
![image](https://user-images.githubusercontent.com/45491456/183433831-ddb35cca-8206-46e8-97c1-e8121b5d2da5.png)
![image](https://user-images.githubusercontent.com/45491456/183433851-122a07ce-1552-4207-877e-4829670f4479.png)
Looks like things are resolved although I didn't check all the themes.
